### DISCUSSION TITLE
Ollama models stop showing after upgrade to v0.1.123

### DISCUSSION THREAD

**Discussion by llagerlof** [open-webui/open-webui#2054] [General]
# Bug Report

## Description

**Bug Summary:**
After upgrade to v0.1.123 the model isn't showing in "Select a model" combo box anymore.

**Steps to Reproduce:**

1. Start the container on Windows 11 using `cmd`, normal user (not admin).

2. `docker run -d -p 3000:8080 -e OLLAMA_BASE_URL=http://127.0.0.1:11434 -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main`

3. Access `http://localhost:3000`, log in and click on "Select a model" combo box field.

**Expected Behavior:**
When opening the "Select a model" field the models available in ollama should appear.

**Actual Behavior:**
The model isn't showing in "Select a model" combo box.

## Environment

- **Open WebUI Version:** 0.1.123
- **Ollama (if applicable):** 0.1.33
- **Operating System:** Windows 11 / cmd
- **Browser (if applicable):** Any

## Reproduction Details

- `ollama list` shows me the models list.
- `http://127.0.0.1:11434` is accessible using a browser. Returns `Ollama is running`.

**Comment by rdlu** [comment]
I'm having the same. There's one error in the docker logs:

`ERROR:apps.openai.main:Connection error: Cannot connect to host api.openai.com:443 ssl:default [Temporary failure in name resolution]`

I have the openai integration enabled. And my dns is working inside the docker host terminal, nslookup and dig returns the correct ip.

I'm using Docker with a debian 12 host, not windows.

**Comment by llagerlof** [comment]
Just to add info, I **don't** have the openai integration enabled.

**Comment by tjbck** [comment]
Seems like you're using a command that's incorrectly configured, try out some of our preconfigured commands provided here to see if they resolve your issue: https://docs.openwebui.com/getting-started/#quick-start-with-docker- 

Keep us updated!

**Reply by llagerlof** [reply]
You are right. I was using `-e OLLAMA_BASE_URL=http://127.0.0.1:11434` instead of `--add-host=host.docker.internal:host-gateway`.

For future reference, the full fixed container startup command is:

`docker run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main`

Thank you.

**Comment by chrisbward** [comment]
This worked for me, when using the `host` network type;
```
docker run -d -e PORT=3000 --network=host --add-host=host.docker.internal:host-gateway -e OLLAMA_BASE_URL=http://127.0.0.1:11434 -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main
```
